SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,8 +1,12 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
+<%(Callout type="warn" message="This [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
 # SpatialOS Unreal GDK toolbar
 
-The SpatialOS Unreal GDK toolbar is an easy way to run and alter [`spatial` command-line tool](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool) (SpatialOS documentation) commands from inside the Unreal Editor.
+The SpatialOS Unreal GDK toolbar provides several functions required for building and launching your client- and server-workers from inside the Unreal Editor.
+
+## Terms used in this doc
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your game's `.uproject` and source folder (for example, `<ProjectRoot>/ShooterGame/`).  
```

Comment:
@ElleEss @anne-edwards @joshuahuburn do we need to put these in a shared glossary?

Reply:
It might be useful to add these to the [Glossary](https://docs.improbable.io/unreal/pre-alpha/content/glossary) page, if we use them consistently wherever we use them.

Reply:
Making another PR for this

Reply:
This flips between "project" and "game". Suggest "game project" to be clearer as there's a lot of flipping between terms in the docs.

Reply:
Done (in GDoc)